### PR TITLE
Billy/1053 balance header

### LINKED ISSUE #1053: implement header with atom balance
there are a few details to hash out from feedback / comments

screen shot 2018-07-30 at 6 10 00 pm

this component can be seen here: https://zpl.io/bWLnE7e

### PR DESCRIPTION
**Pull request by okwme**
Closes #1053 

Description: 
Added the new header style to the wallet balance page. Needed to make some updates to https://github.com/tendermint/ui/pull/62 as well.

Including:
* settings / preferences in icon bar
* back button icon is most left
* "claim" link doesn't appear if there are no rewards
* incorporates the new "copy to clipboard" proposal #1174 
  * wasn't sure if my colors are right since the dims are done with opactiy settings and wanted to reuse the styles we have in our variables file.

Not sure if this should be merged into our develop branch or if we should be implementing all these larger layout changes into a single branch and then merge them all in at once for consistency in style.

Using the new header on other pages should follow this format in front of the menu slot:
```
tm-page
  template(slot="menu-body"): tm-balance(:unstakedAtoms="unstakedAtoms", totalRewards="100")
  div(slot="menu")
    vm-tool-bar
      a(@click='connected && updateBalances()' v-tooltip.bottom="'Refresh'" :disabled="!connected")
        i.material-icons refresh
      a(@click='setSearch()' v-tooltip.bottom="'Search'" :disabled="!somethingToSearch")
        i.material-icons search
```
Where `unstakedAtoms`, `totalRewards` & `totalEarnings` are strings that only show if they're truthy.

also update TmToolBar to consistent naming of VmToolBar do differentiate from the ui library TmToolBar

### CODE REVIEW

**Comment by okwme** [comment]
screenshot 2018-08-28 14 33 42
screenshot 2018-08-28 14 33 48

**Comment by okwme** [comment]
e2es failing at this screen:


locally they fail because localstorage isn't persisting for the introduction popup

**Comment by okwme** [comment]
locally on the `develop` branch my e2e is failing because:
```
panic: listen tcp 0.0.0.0:26656: bind: address already in use
```
@faboweb could this have to do with the new multiple validator testing setup?

on circleci the log says:
```raven@2.6.0 alert: no DSN provided, error reporting disabled
(node:731) [DEP0016] DeprecationWarning: 'root' is deprecated, use 'global'
Xlib:  extension "RANDR" missing on display ":99".
[731:0828/123843.711219:ERROR:bus.cc(395)] Failed to connect to the bus: Failed to connect to socket /var/run/dbus/system_bus_socket: No such file or directory
[731:0828/123843.781245:ERROR:bus.cc(395)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
mainWindow opened
[731:0828/123843.788360:ERROR:bus.cc(395)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")
Redirecting console output to logfile /home/circleci/project/testArtifacts/cli_home/main.log
Adding new peer: localhost
```

**Comment by okwme** [comment]
add config for disabling feature

**Comment by faboweb** [comment]
What does `Vm` stand for?

File: app/src/renderer/components/common/TmBalance.vue [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      .total-earnings(v-if="totalEarnings")
+        .h3 Total Earnings
+        .h2 {{totalEarnings}}
+      .total-rewards(v-if="totalRewards")
+        .h3 Total Rewards
+        .group
+          .h2 {{totalRewards}}
+          router-link(to="claim") Claim
+    .bottom
+      .address(@click="copy") {{address}}
```
Comment (faboweb) [inline app/src/renderer/components/common/TmBalance.vue]: Should this be it's own component?

File: package.json [hunk 2]
```diff
@@ -98,7 +98,7 @@
     "webpack-dev-server": "2.11.2"
   },
   "dependencies": {
-    "@tendermint/ui": "0.2.20",
+    "@tendermint/ui": "https://github.com/tendermint/ui.git#5493de3544dae62dfd8f8cc565076a2f161ce909",
```
Comment (faboweb) [inline package.json]: Let's wait for a release to merge.

File: test/e2e/common.js [hunk 3]
```diff
@@ -48,7 +48,13 @@ module.exports = {
     await module.exports.openMenu(app)
     // click link
     await app.client.$(`a*=${linkText}`).click()
-    await app.client.waitUntilTextExists(".tm-page-header-title", titleText)
+    try {
+      await app.client.waitUntilTextExists(".tm-page-header-title", titleText)
+    } catch (error) {
+      console.log("failed trying to navigate to " + titleText)
```
Comment (faboweb) [inline test/e2e/common.js]: Can you add a comment on why we need to check for errors here?

File: app/src/renderer/components/common/TmBalance.vue (hunk 1 above)
Comment (okwme) [reply app/src/renderer/components/common/TmBalance.vue]: is it used anywhere else?

Comment (faboweb) [reply app/src/renderer/components/common/TmBalance.vue]: Not yet. OK, we can extract it later.

**Comment by faboweb** [comment]
Just `V` or no prefix?

**Pull request merged**